site/layouts/partials/cookie-consent.html

```diff
@@ -0,0 +1,193 @@
+<style>
+#cookieConsentBanner {
```

Comment:
I would suggest to move the inline CSS/JS code to separate files; that way, our linters can check them.